ifrs17-template/Initialization/InitSystemorphUtils.ipynb

```diff
@@ -0,0 +1,170 @@
+                "\n",
+                "\n<a id='init-to-memory'></a>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!eval-notebook \"InitSystemorphRefDataToMemory\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "using Systemorph.Vertex.DataSource;"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Files by Data Node"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var DataNodeFiles = new Dictionary<string,(string, string)[]>(){{\"ES\", new (string, string)[]{(\"DataNodes/DataNodes_ES.csv\", ImportFormats.DataNode), ",
+                "\n                                                                        (\"DataNodes/DataNodeStates_ES_2020_12.csv\", ImportFormats.DataNodeState), ",
+                "\n                                                                        (\"DataNodes/DataNodeParameters_ES_2020_12.csv\", ImportFormats.DataNodeParameter)}}, ",
+                "\n                                                        {\"FR\", new (string, string)[]{(\"DataNodes/DataNodes_FR.csv\", ImportFormats.DataNode), ",
+                "\n                                                                        (\"DataNodes/DataNodeStates_FR_2020_12.csv\", ImportFormats.DataNodeState), ",
+                "\n                                                                        (\"DataNodes/DataNodeParameters_FR_2020_12.csv\", ImportFormats.DataNodeParameter)}}, ",
+                "\n                                                        {\"CH\", new (string, string)[]{(\"DataNodes/DataNodes_CH.csv\", ImportFormats.DataNode), ",
+                "\n                                                                        (\"DataNodes/DataNodeStates_CH_2020_12.csv\", ImportFormats.DataNodeState),  ",
+                "\n                                                                        (\"DataNodes/DataNodeParameters_CH_2020_12.csv\", ImportFormats.DataNodeParameter)}}};"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var TransactionalFilesByDataNode = new Dictionary<string, (string, string)[]>(){{\"ES\", new (string, string)[]{(\"TransactionalData/Openings_ES_2020_12.csv\", ImportFormats.Opening), ",
```

Comment:
Use named Tuple, it should allow you to avoid Item1 and Item2 in the following code. 
Item1 and Item2 are not nice to read.

Reply:
Done